js/compiler/compiler.go

```diff
@@ -251,7 +251,17 @@ func (c *Compiler) compileImpl(
 			return nil, code, err
 		}
 		// the compatibility mode "decreases" here as we shouldn't transform twice
-		return c.compileImpl(code, filename, wrap, lib.CompatibilityModeBase, state.srcMap)
+		var prg *sobek.Program
+		prg, code, err = c.compileImpl(code, filename, wrap, lib.CompatibilityModeBase, state.srcMap)
+		if err == nil && strings.Contains(src, "module.exports") {
+			c.logger.Warningf(
+				"During the compilation of %q, it has been detected that the file combines ECMAScript modules (ESM) "+
```

Comment:
```suggestion
				"During the compilation of %q, it has been detected that the file %q combines ECMAScript modules (ESM) "+
```
Could help if we write the filename there?

Reply:
It is already mentioned just before the comma, so I don't see why it should be repeated. Especially if the path is long it will make the message huge. And will probably be confusing that we repeat it.